src/ui/components/ExplorerHome/ExplorerHome.js

```diff
@@ -88,77 +99,167 @@ const useStyles = makeStyles((theme) => ({
   arrowInput: {width: "40%", display: "inline-block"},
   pageHeader: {color: theme.palette.text.primary},
+  credCircle: {
+    borderColor: theme.palette.blueish,
+  },
+  grainCircle: {
+    borderColor: theme.palette.warning.main,
+  },
+  participantCircle: {
+    borderColor: theme.palette.scPink,
+  },
+  grainPerCredCircle: {
+    borderColor: theme.palette.green,
+  },
 }));
 
+const CRED_SORT = deepFreeze({
+  name: Symbol("Cred"),
+  fn: (n) => n.cred,
+});
+const GRAIN_SORT = deepFreeze({
+  name: Symbol("Grain"),
+  fn: (n) => n.grainEarned,
+});
+const PAGINATION_OPTIONS = deepFreeze([50, 100, 200]);
+
 type ExplorerHomeProps = {|
   +initialView: CredGrainView | null,
+  +currency: CurrencyDetails,
 |};
 
-export const ExplorerHome = ({initialView}: ExplorerHomeProps): ReactNode => {
+export const ExplorerHome = ({
+  initialView,
+  currency: {suffix: currencySuffix, name: currencyName},
+}: ExplorerHomeProps): ReactNode => {
   if (!initialView) return null;
 
   const classes = useStyles();
   const [tab, setTab] = useState<number>(1);
   const [checkboxes, setCheckboxes] = useState({
-    participants: false,
-    organizations: false,
-    bots: false,
-    projects: false,
+    [IdentityTypes.USER]: false,
+    [IdentityTypes.ORGANIZATION]: false,
+    [IdentityTypes.BOT]: false,
+    [IdentityTypes.PROJECT]: false,
   });
-  const {participants, organizations, bots, projects} = checkboxes;
-
-  const handleChange = (event) => {
-    setCheckboxes({...checkboxes, [event.target.name]: event.target.checked});
-  };
 
-  const data = [
-    {title: "Cred This Week", value: 610},
-    {title: "Grain Harvested", value: "6,765g"},
-    {title: "Active Participants", value: 13},
-    {title: "Grain per Cred", value: "22g"},
-  ];
+  const allParticipants = useMemo(
+    () => Array.from(initialView.participants()),
+    [initialView.participants()]
+  );
 
-  const createData = (username, cred, grain, chart) => ({
-    username,
-    cred,
-    grain,
-    chart,
+  const tsParticipants = useTableState(allParticipants, {
+    initialRowsPerPage: PAGINATION_OPTIONS[0],
+    initialSort: {
+      sortName: CRED_SORT.name,
+      sortOrder: SortOrders.DESC,
+      sortFn: CRED_SORT.fn,
+    },
   });
 
-  // sort by cred amount, highest to lowest
-  const nodes = sortBy(initialView.participants(), (n) => -n.cred);
-  // create an array of 0s for the cred summary graph at the top of the page
-  let credTimelineSummary = initialView.intervals().map(() => 0);
+  const {credTimelineSummary, credAndGrainSummary} = useMemo(() => {
+    let credTimelineAggregator = [];
+    const credAndGrainAggregator = {
+      totalCred: 0,
+      totalGrain: fromInteger(0),
+      avgCred: 0,
+      avgGrain: fromInteger(0),
+    };
 
-  const rows = nodes.map((node) => {
-    const {credPerInterval} = node;
+    if (tsParticipants.currentPage.length > 0) {
+      for (const participant of tsParticipants.currentPage) {
```

Comment:
(we will have to add a new `allPages` output to TableState to implement my aggregation across all pages, it appears)